core/src/main/java/jenkins/model/ModelObjectWithContextMenu.java

```diff
@@ -255,7 +255,7 @@ public ContextMenu add(Node n) {
          *
          * @since 1.513
          */
-        public ContextMenu add(Computer c) {
+        public ContextMenu add(IComputer c) {
```

Comment:
Not binary-compatible (I think). Are there potentially any external callers?

Reply:
I checked but doesn't seem likely. I've retained the previous signature just in case.